modules/calendar.xml

```diff
@@ -128,5 +128,35 @@ if(typeof ongoingEvent != 'undefined' && ongoingEvent && properties.currentEvent
 // Start effects for this widget
 $(target).xiboLayoutAnimate(properties);
     ]]></onVisible>
+    <sampleData><![CDATA[
+[{
+    "summary": "Sample 1",
+    "description": "Something here",
```

Comment:
Can we have a go at making some better summaries/descriptions for these events? Even if its just "Sample Event 1" and the description explaining what we're doing with the dates.

Reply:
It looks great, please go ahead!! :+1: